samples: fast_pair: locator_tag: Switch to NCS_MCUBOOT_DISABLE_SELF_RWX (#28746)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

* :ref:`fast_pair_locator_tag` sample:

  * Updated:

    * UI thread handling for reference board targets with a speaker by moving the speaker control into the common indication thread.
      The signaling is now done using the :ref:`zephyr:events` API.

>     * MCUboot's the configuration of the non-volatile memory self-protection mechanism in the MCUboot image on the nRF54L board targets.

Reviewer: ```suggestion
    * MCUboot's configuration of the non-volatile memory self-protection mechanism in the MCUboot image on the nRF54L board targets.
```

>     * MCUboot's the configuration of the non-volatile memory self-protection mechanism in the MCUboot image on the nRF54L board targets.

Reviewer 2: ```suggestion
    * The configuration of the non-volatile memory self-protection mechanism in the MCUboot image on the nRF54L board targets.
```

"in the MCuboot image" clearly indicates that is MCUBoot configuration - no need to double the phrase

>       The :kconfig:option:`CONFIG_NCS_MCUBOOT_DISABLE_SELF_RWX` Kconfig option now replaces the :kconfig:option:`CONFIG_FPROTECT`, which is associated with the :ref:`fprotect_readme` library.

Reviewer 2: why did you remove the last sentence?

The new mechanism uses a dedicated RRAMC region to disable read, write, and execute access to the MCUboot partition right before jumping to the application image.

seems like a nice explanation

* :ref:`fast_pair_locator_tag` sample:

  * Updated:

    * UI thread handling for reference board targets with a speaker by moving the speaker control into the common indication thread.
      The signaling is now done using the :ref:`zephyr:events` API.

> 

Reviewer 2: I think a blank line is just needed between the different categories of changes "Fixed/Updated/Added", but when the category is the same you can omit the blank lines

  * Fixed the ringing status indications with the green LED flashes for reference board targets.
    A ringing status indication was often skipped during the motion detection event.

>   * Updated MCUboot's self-protection on the nRF54L board targets to use the :kconfig:option:`CONFIG_NCS_MCUBOOT_DISABLE_SELF_RWX` Kconfig option instead of the :ref:`fprotect_readme` library.

Reviewer 2: ~Please merge with the Updated syntax at the top of the Fast Pair samples section~

>   * Updated MCUboot's self-protection on the nRF54L board targets to use the :kconfig:option:`CONFIG_NCS_MCUBOOT_DISABLE_SELF_RWX` Kconfig option instead of the :ref:`fprotect_readme` library.

Reviewer 2: ```suggestion
  * Updated the configuration of the non-volatile memory self-protection mechanism in the MCUboot image on the nRF54L board targets.
    The :kconfig:option:`CONFIG_NCS_MCUBOOT_DISABLE_SELF_RWX` Kconfig option now replaces the :kconfig:option:`CONFIG_FPROTECT`, which is associated with the :ref:`fprotect_readme` library.
```

>     The new mechanism uses a dedicated RRAMC region to disable read, write, and execute access to the MCUboot partition right before jumping to the application image.

Reviewer: ```suggestion
  * Updated:
 
    * UI thread handling for reference board targets with a speaker by moving the speaker control into the common indication thread.
      The signaling is now done using the :ref:`zephyr:events` API.

    * MCUboot's self-protection on the nRF54L board targets to use the :kconfig:option:`CONFIG_NCS_MCUBOOT_DISABLE_SELF_RWX` Kconfig option instead of the :ref:`fprotect_readme` library.
      The new mechanism uses a dedicated RRAMC region to disable read, write, and execute access to the MCUboot partition right before jumping to the application image.

  * Fixed the ringing status indications with the green LED flashes for reference board targets.
    A ringing status indication was often skipped during the motion detection event.
```
